polkadot/node/network/approval-distribution/src/lib.rs

```diff
@@ -2519,23 +2731,31 @@ impl ApprovalDistribution {
 	pub async fn handle_from_orchestra<
 		N: overseer::SubsystemSender<NetworkBridgeTxMessage>,
 		A: overseer::SubsystemSender<ApprovalVotingMessage>,
+		RA: overseer::SubsystemSender<RuntimeApiMessage>,
```

Comment:
We can not merge them into the same type and variable, because down the line in the follow up PRs: https://github.com/paritytech/polkadot-sdk/pull/4849#issue-2364261568 we wouldn't be able to inject different senders.